doc/ledger3.texi

```diff
@@ -6430,6 +6433,9 @@ Aliases are completely ignored.
 @item --no-color
 Suppress any color TTY output.
 
+@item --no-pager
+Direct output to stdout, avoiding pager program.
+
```

Comment:
Craig before me, and me sometimes ago, we add a thought on how to restructure the documentation. I have to admit, that me, discovering the size of the task and the difficulty, somewhat procrastinate to do it.
Anyway, I have some thoughts about restructuration. Yes, this is more than cumbersome to document in several places. But you can not list all options without a introduction. The balance is hard to maintain.
About the links, what you propose is HTML URL minded. What is currently implemented is based on another point of view : any item in documentation is added to index, and you refer to index to jump to relevant section. More than a book. At the moment, I have no prefered option. I am just used to use the second one. And this make the PDF more readable. But I agree that HTML linking is helpfull too. It is just ugly in the PDF outputed from texinfo files.